README.md

```diff
@@ -32,16 +32,11 @@ export default class MyComponent extends Component {
 }
 ```
 
-#### UMD module:
```

Comment:
hey! I'm not sure why we removed this. Is the file not available in the npm package?